src/system/stacktrace.c

```diff
@@ -3,7 +3,11 @@
 #ifdef __GNUC__
 #include <execinfo.h>
 #endif
-#include <unistd.h>
+#ifdef WINDOWS_OS
+    #include <io.h>
+#else
+    #include <unistd.h>
```

Comment:
The macro should not work on Windows anyway. Please put `unistd.h` under `#ifdef __GNU__`.